docs/index.html

```diff
@@ -152,7 +152,7 @@ <h4>Desktop</h4>
             <li>Firefox</li>
             <li>Safari 5+</li>
             <li>Opera 12+</li>
-            <li>IE 7&ndash;11</li>
+            <li>IE 9&ndash;11</li>
```

Comment:
No support too

Reply:
This specific PR didn't drop support for it, I'm just keeping things narrowly scoped. I think we'll overhaul the browser support section anyway when pushing on v2.